### PR TITLE
Small cleanup

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

File: src/analysis/bounds.rs [hunk 1]
```diff
@@ -49,18 +49,19 @@ impl Bounds {
             _ => None,
         }
     }
-    pub fn to_glib_extra(library: &Library, type_id: TypeId, is_nullable: bool) -> String {
+    pub fn to_glib_extra(library: &Library, type_id: TypeId,
+                         is_nullable: Nullable) -> String {
```
Comment (EPashkin) [inline src/analysis/bounds.rs]: Just `nullable: Nullable` pls

Comment (GuillaumeGomez) [inline src/analysis/bounds.rs]: Yes sir!

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by EPashkin** [comment]
Thanks, merge when it builds, please. I off.

File: src/analysis/bounds.rs [hunk 2]
```diff
@@ -49,18 +49,19 @@ impl Bounds {
...
             _ => String::new(),
         }
     }
     pub fn add_parameter(&mut self, name: &str, type_str: &str, bound_type: BoundType,
-                         is_nullable: bool) -> bool {
+                         nullable: Nullable) -> bool {
         if self.used.iter().any(|ref n| n.0 == name) { return false; }
         if bound_type == BoundType::Into {
-            if is_nullable == false { return true; }
+            if *nullable == false { return true; }
```
Comment (EPashkin) [inline src/analysis/bounds.rs]: Not simpler `!*nullable`?

Comment (GuillaumeGomez) [inline src/analysis/bounds.rs]: I like being explicit. But if it bothers you, I can rewrite it this way.

Comment (EPashkin) [inline src/analysis/bounds.rs]: NP

**Comment by GuillaumeGomez** [comment]
Since you said it was ok, I merge it. Thanks for your review @EPashkin!

**Pull request merged**